app/Http/Controllers/Auth/LoginController.php

```diff
@@ -44,7 +44,13 @@ class LoginController extends AuthController
      */
     protected function redirectTo()
     {
-        $redirectTo = WhichPortal::isManagerPortal() ? route('manager.home') : route('home');
+        if (WhichPortal::isManagerPortal()) {
+            $redirectTo = WhichPortal::isManagerPortal() ? route('manager.home') : route('home');
```

Comment:
The ternary operator is unnecessary here, since we're now doing the check in the if statement.